src/Instrumentation/HttpAsyncClient/_register.php

```diff
@@ -2,4 +2,11 @@
 
 declare(strict_types=1);
 
-\OpenTelemetry\Contrib\Instrumentation\HttpAsyncClient\HttpAsyncClientInstrumentation::register();
+use OpenTelemetry\Contrib\Instrumentation\HttpAsyncClient\HttpAsyncClientInstrumentation;
+use OpenTelemetry\SDK\Sdk;
+
+if (Sdk::isInstrumentationDisabled('http-async-client') === true) {
```

Comment:
According to spec, libraries should not assume that an SDK is installed: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/overview.md#sdk, so this might also need a further check (I thought `class_exists` would be enough)

Reply:
What would be the logic if the SDK is _not_ installed? 
1. Don't run any code from the `_register.php` files (early return)
2. Do run the code, but don't do this check, and assume that instrumentation is enabled
3. Run any other content in this file, but assume instrumentation is disabled

Reply:
I have implemented a `class_exist` check inline, and ignore the check if we don't have the SDK library